pkg/kubelet/cm/helpers_linux.go

```diff
@@ -44,16 +44,16 @@ const (
 	SharesPerCPU  = 1024
 	MilliCPUToCPU = 1000
 
-	// 100000 is equivalent to 100usec
-	QuotaPeriod    = 100000
-	MinQuotaPeriod = 1000
+	// 100000 microseconds is equivalent to 100ms
+	QuotaPeriod    = 100000 // in microseconds
```

Comment:
I think we can also drop all over the place the `// in microseconds`
The change in line 47 seems to be enough to clarify it.

Reply:
I prefer to leave it like that if we don't rename it to `QuotaPeriodMicroSeconds`. These two are non-obviously have units, and I want everyone to see that comment each time someone hovers over one of these in Goland.
<img width="463" alt="image" src="https://user-images.githubusercontent.com/712534/187974393-ed40e41b-5630-454d-8357-b766ccd64d62.png">

Reply:
<img width="433" alt="image" src="https://user-images.githubusercontent.com/712534/187974574-237f01bc-2076-4cde-9c67-6819bebce2ac.png">
The first one has both comments, which is a little cumbersome — I'll remove the comment only from it.

Reply:
I've altered the comment on MinQuotaPeriod to match the existing comment on QuotaPeriod.